components/Btn.tsx

```diff
@@ -0,0 +1,55 @@
+import { ButtonHTMLAttributes, useMemo } from "react";
+import Image from "next/image";
+import ic_plus_black from "@/assets/icons/plus_black.svg";
+import ic_plus_white from "@/assets/icons/plus_white.svg";
+import ic_cross from "@/assets/icons/cross.svg";
+import ic_check from "@/assets/icons/check.svg";
+import styles from "./Btn.module.css";
+
+interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
+  size?: "large" | "small";
+  mode: "add" | "delete" | "edit";
+}
+
+const CONTENT: Record<Props["mode"], string> = {
+  add: "추가하기",
+  delete: "삭제하기",
+  edit: "수정 완료",
+};
+
+export default function Btn({
+  className = "",
+  size = "large",
+  mode,
+  ...props
+}: Props) {
+  const img = useMemo(() => {
+    switch (mode) {
+      case "add":
+        return { alt: "plus", src: ic_plus_black };
+      case "delete":
+        return { alt: "cross", src: ic_cross };
+      case "edit":
+        return { alt: "check", src: ic_check };
+    }
+  }, [mode]);
+
+  return (
+    <button className={`${styles.btn} ${styles[size]} ${className}`} {...props}>
```

Comment:
💊 제안
type이 명시되지 않으면 submit 이므로 가능한 button type을 기본값으로 가질 수 있게 해주시는 것을 추천드립니다.

```suggestion
  return (
    <button className={`${styles.btn} ${styles[size]} ${className}`} type={type ?? 'button'}{...props}>
```